.github/actions/yarn-nm-install/action.yml

```diff
@@ -0,0 +1,65 @@
+########################################################################################
+# "yarn install" composite action for yarn 3/4+ and "nodeLinker: node-modules"         #
+#--------------------------------------------------------------------------------------#
+# References:                                                                          #
+#   - bench: https://gist.github.com/belgattitude/0ecd26155b47e7be1be6163ecfbb0f0b     #
+########################################################################################
+
+name: 'Monorepo install (yarn)'
+description: 'Run yarn install with node_modules linker and cache enabled'
+inputs:
+  enable-corepack:
+    description: 'Enable corepack'
+    required: false
+    default: 'false'
+  cache-install-state:
+    description: 'Cache yarn install state (invalidated on lock changes)'
+    required: false
+    default: 'true'
+  cache-node-modules:
+    description: 'Cache node_modules (invalidated on lock changes)'
+    required: false
+    default: 'false'
```

Comment:
I let the default to false. But in the strapi workflow it might make sense to enable it. Cost/benefit... Need to measure with default: 'true' and default 'false'